Automattic/wp-calypso
# My Home: Try Home Layout i1.

This PR explores what changes we need to make for something like My Home Layout i1.

See: pbAPfg-g0-p2

#### Testing instructions
* Apply D41677-code and sandbox the API.
* Go to `/home/:site` with an prelaunch, launched and established site and observe the layout.
